.circleci/config.yml

```diff
@@ -60,28 +60,19 @@ aliases:
         only: /^release\/.*\.0$/
 
 commands:
-  install-and-create-sim:
+  install-runtime:
     parameters:
-      install-name:
-        type: string
-      sim-device-type:
-        type: string
-      sim-device-runtime:
-        type: string
-      sim-name:
+      runtime-name:
         type: string
     steps:
-      - run:
-          name: Install xcode-install
-          command: gem install xcode-install
       - run:
           name: Install simulator
           command: | # Print all available simulators and install required one
-              xcversion simulators
-              xcversion simulators --install="<< parameters.install-name >>"
+              xcodes runtimes
+              sudo xcodes runtimes install "<< parameters.runtime-name >>"
       - run:
-          name: Create simulator
-          command: xcrun simctl create '<< parameters.sim-name >>' com.apple.CoreSimulator.SimDeviceType.<< parameters.sim-device-type >> com.apple.CoreSimulator.SimRuntime.<< parameters.sim-device-runtime >>
+          name: List simulators
+          command: xcrun simctl list
```

Comment:
Hmm I was wondering, do we need to list the simulators here?

Reply:
Oh thanks! This was for debugging. Let me remove